lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid;
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  max-width: 860px;
+  width: fit-content;
```

Comment:
How well did you test this across browsers?  I always have trouble with it, given that the specs have gone back and forth over whether it should be a keyword or a function (e.g. `fit-content(100px)`).  Looks like the way it's turned out is CSS 3 only defines the function syntax and the keyword syntax is deferred to CSS 4.

Moreover, I don't think the content of this component (an icon followed by non-wrapping text) has any difference between the min and max widths, so `min-content`, `max-content,` and `fit-content` all end up doing the same thing, and either of the other two are simpler to understand and probably more widely supported.

Reply:
> Moreover, I don't think the content of this component (an icon followed by non-wrapping text) has any difference between the min and max widths, so min-content, max-content, and fit-content all end up doing the same thing, and either of the other two are simpler to understand and probably more widely supported.

This does beg the question of what should happen when the indicator wants to be wider than its container.  Right now it overflows. Should it use ellipsis truncation?

Reply:
> > Moreover, I don't think the content of this component (an icon followed by non-wrapping text) has any difference between the min and max widths, so min-content, max-content, and fit-content all end up doing the same thing, and either of the other two are simpler to understand and probably more widely supported.
> 
> This does beg the question of what should happen when the indicator wants to be wider than its container. Right now it overflows. Should it use ellipsis truncation?

Truncation: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/3da1f0356c771ee48cfe2d33ec036362524de1c8

Reply:
> How well did you test this across browsers? I always have trouble with it, given that the specs have gone back and forth over whether it should be a keyword or a function (e.g. `fit-content(100px)`). Looks like the way it's turned out is CSS 3 only defines the function syntax and the keyword syntax is deferred to CSS 4.
> 
> Moreover, I don't think the content of this component (an icon followed by non-wrapping text) has any difference between the min and max widths, so `min-content`, `max-content,` and `fit-content` all end up doing the same thing, and either of the other two are simpler to understand and probably more widely supported.

I used `min-content` in the end: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/3da1f0356c771ee48cfe2d33ec036362524de1c8

Reply:
> > How well did you test this across browsers? I always have trouble with it, given that the specs have gone back and forth over whether it should be a keyword or a function (e.g. `fit-content(100px)`). Looks like the way it's turned out is CSS 3 only defines the function syntax and the keyword syntax is deferred to CSS 4.
> > Moreover, I don't think the content of this component (an icon followed by non-wrapping text) has any difference between the min and max widths, so `min-content`, `max-content,` and `fit-content` all end up doing the same thing, and either of the other two are simpler to understand and probably more widely supported.
> 
> I used `min-content` in the end: [3da1f03](https://github.com/sonatype/sonatype-react-shared-components/commit/3da1f0356c771ee48cfe2d33ec036362524de1c8)

And then I read the spec some more and decided that `max-content` was better: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/694452a627541181b8abcb3a14fa5eec174586db